valhalla/userrequests/filters.py

```diff
@@ -9,6 +9,7 @@ class UserRequestFilter(filters.FilterSet):
     state = django_filters.ChoiceFilter(choices=UserRequest.STATE_CHOICES)
     title = django_filters.CharFilter(name='group_id', lookup_expr='icontains', label='Title contains')
     user = django_filters.CharFilter(name='submitter__username', lookup_expr='icontains', label='Username contains')
+    exclude_state = django_filters.CharFilter(name='state', lookup_expr='icontains', label='Exclude State', exclude=True)
```

Comment:
Why don't you just negate the already existing state filter (line 9) with `exclude=True` making it a freeform field with icontains will just slow it down.